packages/mux-player/src/errors.ts

```diff
@@ -68,8 +68,8 @@ export function getErrorLogs(
             dialog.message = i18n(`The video’s secured playback-token has expired.`, translate);
             devlog.message = i18n(`The video’s secured playback-token has expired.`, translate);
             devlog.context = i18n(`Expired at: {expiredDate}. Current time: {currentDate}.`, translate).format({
-              expiredDate: new Intl.DateTimeFormat(lang.code, dateOptions).format(tokenExpiry * 1000),
-              currentDate: new Intl.DateTimeFormat(lang.code, dateOptions).format(Date.now()),
+              expiredDate: new Intl.DateTimeFormat((lang as any).code, dateOptions).format(tokenExpiry * 1000),
```

Comment:
NOTE: These and similar are due to slight changes in type checking expectations with TypeScript 5.5 vs. 4.9